src/test/scala/ai/privado/languageEngine/go/tagger/GoTaggingTestBase.scala

```diff
@@ -80,8 +83,53 @@ abstract class GoTaggingTestBase extends AnyWordSpec with Matchers with BeforeAn
     )
   )
 
+  val sinkRule = List(
+    RuleInfo(
+      Constants.thirdPartiesAPIRuleId,
+      "Third Party API",
+      "",
+      Array(),
+      List(
+        "((?i)((?:http:|https:|ftp:|ssh:|udp:|wss:){0,1}(\\/){0,2}[a-zA-Z0-9_-][^)\\/(#|,!>\\s]{1,50}\\.(?:com|net|org|de|in|uk|us|io|gov|cn|ml|ai|ly|dev|cloud|me|icu|ru|info|top|tk|tr|cn|ga|cf|nl)).*(?<!png|jpeg|jpg|txt|blob|css|html|js|svg))"
+      ),
+      false,
+      "",
+      Map(),
+      NodeType.API,
+      "",
+      CatLevelOne.SINKS,
+      catLevelTwo = Constants.third_parties,
+      Language.JAVASCRIPT,
```

Comment:
Lang to Go

Reply:
Addressed here: https://github.com/Privado-Inc/privado-core/pull/779/commits/4dc0d5ae6dba8ffc3b56b0dd6d716316799fa167